pkg/gds/admin/v2/client_test.go

```diff
@@ -165,6 +165,34 @@ func TestListVASPs(t *testing.T) {
 	require.Equal(t, fixture.Count, out.Count)
 }
 
+func TestRetrieveVASP(t *testing.T) {
+	fixture := &admin.RetrieveVASPReply{}
```

Comment:
I think it's a good idea to just use this dummy reply here, since an actual VASP takes up so many lines. Maybe before this line we could have an inline comment with a tinyurl link to the VASP definition from the protocol buffers: https://github.com/trisacrypto/trisa/blob/main/proto/trisa/gds/models/v1beta1/models.proto#L20-L91